src/manifest.ts

```diff
@@ -64,6 +65,8 @@ export class PluginsUpdater implements ObsidianManifest {
         this.totalPlugins = 0;
         this.checkedPlugins = 0;
 
+        this.progressBar = new ProgressBar(plugin, "plugin-updating", this.totalPlugins);
+        /*
         this.noticeEl = document.createDocumentFragment();
```

Comment:
remove the useless code other than comment it

Reply:
fixed by [fff37fe](https://github.com/edonyzpc/personal-assistant/pull/97/commits/fff37fe7db4688e939e9ea12aaccd262523c0191)